VocaDbWeb/Views/Shared/Partials/_SongListsFilters.cshtml

```diff
@@ -1,22 +1,43 @@
-﻿@using Res = ViewRes.SongList.FeaturedStrings;
-
-<div class="clearfix song-lists-filters">
-	<div class="pull-left">
-		<span>@Res.SortBy</span>
-		<div class="btn-group">
-			<a data-bind="click: function() { sort('Date'); }, css: { active: sort() == 'Date' }" visible="{{ showEventDateSort }}" class="btn" href="#">
-				@VocaDb.Web.Resources.Views.SongList.SongListSortRuleNames.Date
-			</a>
-			<a data-bind="click: function() { sort('CreateDate'); }, css: { active: sort() == 'CreateDate' }" class="btn" href="#">
-				@VocaDb.Web.Resources.Views.SongList.SongListSortRuleNames.CreateDate
-			</a>
-			<a data-bind="click: function() { sort('Name'); }, css: { active: sort() == 'Name' }" class="btn" href="#">
-				@VocaDb.Web.Resources.Views.SongList.SongListSortRuleNames.Name
-			</a>
+﻿<div class="form-horizontal well well-transparent" data-bind="visible: true" style="display: none;">
+	
+	<div class="pull-right">

```

Comment:
I don't know why did you change this to dropdown instead of tabs, but I guess it works like this too.

Reply:
Yes, and I think this is more consistent with other layouts.